### DISCUSSION TITLE
Thermostat off despite target temp not reached

### DISCUSSION THREAD

**Discussion by av13ebuy** [jmcollin78/versatile_thermostat#647] [Q&A]
Hello

I am a newbie and I am a bit lost.
I have installed this thermostat, however I do not understand how it works.

I have a temperature of the room in the picture at 19.4C.
I have requested 19.9C but my thermostat does not make it on to reach the target temperature.
Can someone explain me the "reason"?

thank you for your help :)

![image](https://github.com/user-attachments/assets/252e7f3d-ca32-4612-a867-fcf3d0327e1f)

**Comment by jmcollin78** [comment]
Hello, please read this: https://github.com/jmcollin78/versatile_thermostat/blob/main/README.md#algorithm
